READMEs/env-vars.md

```diff
@@ -0,0 +1,31 @@
+# Environment Variables
```

Comment:
Feedback:

1. Please call this file envvars.md, not env-vars.md. Less annoying for us to work with.

2. Please link to this from README.md, in the bottom section.

3. Please link to this from each agent-specific README; from the backend-dev README; and from the frontend-dev README. Keep it concise.

4. There actually _is_ a bit of documentation on the filter envvars, at the bottom of subgraph.md. To reconcile (and avoid DRY): in subgraph.md, just mention the 4 envvars, and then link to envvars.md.